tools/affected-apps

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
why not have `.sh` extension?

Reply:
also should make it `set-affected-apps-env.sh`

Reply:
It's just my preference. I think this looks better.

Reply:
But I do agree about the naming

Reply:
it's very confusing about what the file is, without looking at its content

Reply:
IMO it's pretty clear what it is from just the name

Reply:
@paphonb the problem is i don't know that it's an executable, i thought it's just a text file, nothing else